### ISSUE TITLE
Unit's .toNumeric() incorrect immediately after calculation with mixed units

### ISSUE THREAD

**Issue by AlexanderBeyn** [josdejong/mathjs#901]
When doing calculations with mixed units, such as multiplying feet and meters, .toNumeric() returns an incorrect value immediately after the calculation. Once .formatUnits() has been called on the result, .toNumeric() returns the correct value.  

An example at http://jsbin.com/wusavigugi/edit?html,js,output shows that 
```
    const a = math.unit(1, 'ft').multiply(math.unit(1, 'meter'))
    document.write(a.toNumeric() + '<br>');
    document.write(a.formatUnits() + '<br>');
    document.write(a.toNumeric() + '<br>');
```

outputs
```
1
meter^2
0.3048
```

**Comment by ericman314** [comment]
Thanks for reporting. It seems like this is due to the lazy simplification of units. Combining and simplifying all the units accrued during an operation is costly, so we only do it at the end when the units is formatted as a string. Perhaps we should alter `toNumeric` to also trigger the simplification. I can get to this myself next week, or you can submit a PR yourself if it is urgent.

**Comment by josdejong** [comment]
Fixed via #902 (fix not yet released at this moment)

**Comment by josdejong** [comment]
Should be fixed now in `v3.16.1`
